scripts/partition_manager.py

```diff
@@ -540,7 +540,10 @@ def solve_simple_region(pm_config, start, size, placement_strategy, region_name,
         address = start + size - reserved
     else:
         address = start + reserved
-    for partition_name in pm_config:
+
+    # Static partitions are now added to the pm_config dict. These partitions
+    # already have their 'address' set, so skip these partitions in this loop.
```

Comment:
This is because they are already factored in via `reserved`, right?

Reply:
Correct